test/templates/table/component.njk

```diff
@@ -41,29 +41,29 @@ table:
 <div class="fd-table--fixed">
 {% endif %}
 <table  class="fd-table{{ modifier.block | modifier('table') }}{{ state | state }}{{ classes | classes }}"{{ aria | aria }}>
-    <thead>
-        <tr>
+    <thead class="fd-table__header">
+        <tr class="fd-table__row">
             {%- for header in properties.headers %}
             {%- if header.context_menu -%}
-            <th class="fd-table__context-menu {{header.classes | classes}}" style="left:{{0 if header.fixed_col else header.offset}}; width:{{header.width}}" {{header.aria | aria}}  scope="col" >
+            <th class="fd-table__context-menu {{header.classes | classes}} fd-table__cell " style="left:{{0 if header.fixed_col else header.offset}}; width:{{header.width}}" {{header.aria | aria}}  scope="col" >
```

Comment:
Should this have inline styles? I think that defeats the purpose of testing the styles.